src/types/index.d.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable camelcase */
+import type { GridLayerOptions, Coords, DoneCallback } from "leaflet";
+
+export type PixelValueToColorFn = (values: number[]) => string;
```

Comment:
I think there's a slight typo here and it should be PixelValue**s**ToColorFn?

Reply:
Yup, great catch.